src/object.cr

```diff
@@ -1072,6 +1072,7 @@ class Object
   #   delegate downcase, to: @string
   #   delegate gsub, to: @string
   #   delegate empty?, capitalize, to: @string
+  #   delegate :[], to @string
```

Comment:
Oh but I think you are missing the **:** here. At the other lines its "to: @string" at the end. But here its "to @string".